### DISCUSSION TITLE
[FR] Add notes to invoices

### DISCUSSION THREAD

**Discussion by mikestecker** [craftcms/console#90] [Craft Console]
It would be nice to be able to add notes to invoices like we can with licenses.

**Comment by davidhellmann** [comment]
Oh, we need that too. A more advanced option would be Tagging <3

**Comment by janhenckens** [comment]
I'd like to bump this one up again, as requested by the people that handle our invoicing.

It would be great if we could add a note during purchasing and if that note could show up on the invoice as well. That way it's much much easier to link invoices to projects/clients.

**Comment by giantoctopus** [comment]
Hi all - it's 2022 and this still is a priority for invoicing - currently there are no distinguishing notes saying who the invoice is for if you are an Agency handling this for clients.
